collectors/pool_usage.go

```diff
@@ -55,6 +59,15 @@ func NewPoolUsageCollector(conn Conn) *PoolUsageCollector {
 			},
 			[]string{"pool"},
 		),
+		MaxAvail: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: cephNamespace,
+				Subsystem: "df",
```

Comment:
Sure; I didn't quite expect you to be open for radical metric renamings. I'd propose to keep them named the way ceph names them; unless such a choice would be unclear.

You haven't used 'ceph pg dump pools --format=json-pretty'; You should try it. Most of the counters you create in health.go and pool_usage.go are in there too; The only thing missing in that output is the bytes available for each pool.

Shall I create a PR that exposes most of those counters; then see what is redundant and cut that away?